rem/aufile/aufile.c

```diff
@@ -278,3 +278,44 @@ size_t aufile_get_length(struct aufile *af, struct aufile_prm *prm)
 
 	return 0;
 }
+
+/**
+ * Set initial playing position of a WAV file in ms
+ *
+ * @param af  Audio-file
+ * @param prm Audio file parameters from aufile_open
+ *
+ * @return position in bytes or (size_t)-1 in case of an error.
+ */
+size_t aufile_set_position(struct aufile *af, struct aufile_prm *prm,
```

Comment:
- return int as error code
- correct  @return doxygen

Reply:
Done

Reply:
`int` instead of `size_t`